packages/eas-cli/src/commands/build/version/get.ts

```diff
@@ -0,0 +1,118 @@
+import { Platform } from '@expo/eas-build-job';
+import { EasJsonAccessor } from '@expo/eas-json';
+import { Flags } from '@oclif/core';
+import chalk from 'chalk';
+
+import EasCommand from '../../../commandUtils/EasCommand';
+import { EasNonInteractiveAndJsonFlags } from '../../../commandUtils/flags';
+import { AppVersionQuery } from '../../../graphql/queries/AppVersionQuery';
+import { toAppPlatform } from '../../../graphql/types/AppPlatform';
+import Log from '../../../log';
+import { selectRequestedPlatformAsync, toPlatforms } from '../../../platform';
+import { getApplicationIdentifierAsync } from '../../../project/applicationIdentifier';
+import {
+  ensureVersionSourceIsRemoteAsync,
+  validateAppConfigForRemoteVersionSource,
+} from '../../../project/remoteVersionSource';
+import { enableJsonOutput, printJsonOnlyOutput } from '../../../utils/json';
+import { getProfilesAsync } from '../../../utils/profiles';
+
+export default class BuildVersionGetView extends EasCommand {
+  public static override description = 'Return version stored on EAS servers';
```

Comment:
So it matches the command name:
```suggestion
  public static override description = 'Get the latest version from EAS servers.';
```